db/seeds.rb

```diff
@@ -1,7 +1,15 @@
-# This file should contain all the record creation needed to seed the database with its default values.
+# This file contains the record creation calls needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
+ApplicationRecord.transaction do
+  PreservationPolicy.seed_from_config
+  EndpointType.seed_from_config
+  Endpoint.seed_local_online_endpoints_from_config
+  Status.seed_from_config
+end
```

Comment:
so is the idea that we will call this via rake db:xxx task for all/any of the environments?  Should there be a comment that it can be safely run repeatedly?

Reply:
yup, `rake db:seed` should be callable from where-ever, and it should be safe to run repeatedly.  good call on adding that as a note, will do...